chall08/sluhtala.c

```diff
@@ -0,0 +1,25 @@
+int ft_ie_except_after_c(char *str)
+{
+	int i;
+
+	i = 0;
+	if (str == NULL)
+		return (-1);
+	if (str[0] == '\0')
+		return (1);
+	if (str[0] == 'e' && str[1] == 'i')
+		return (0);
+	while (str[i])
+	{
+		if (str[i] != 'c' && str[i + 1] == 'e' && str[i + 2] == 'i')
+		{
+			return (0);				
+		}
+		if (str[i] == 'c' && str[i + 1] == 'i' && str[i + 2] == 'e')
+		{
+			return (0);
+		}
```

Comment:
A good way to check the letter rule. Except you don't check if you are going over the array, so that will cause some crashes with some cases.

You also previously didn't have brackets with a single line if-blocks so having them here is a bit strange. Try to stick to one format.

Reply:
Just commenting that if statement won't go over the array, because it stops if (str[i + 1]) finds null terminator so it won't check (str[i + 2]). If there is no null terminator while loops wouldn't stop anyway so we have to assume that only null terminated strings are given to the function, otherwise we would need to receive the length of the char array too.